sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessErrorEventArgs.cs

```diff
@@ -37,12 +40,12 @@ public ProcessErrorEventArgs(Exception exception, ExceptionReceivedEventArgsActi
         public ExceptionReceivedEventArgsAction Action { get; }
 
         /// <summary>The namespace name used when this exception occurred.</summary>
-        public string Endpoint { get; }
+        public string FullyQualifiedNamespace { get; }
 
         /// <summary>The entity path used when this exception occurred.</summary>
-        public string EntityPath { get; }
+        public string EntityName { get; }
 
         /// <summary>The Client Id associated with the sender, receiver or session when this exception occurred.</summary>
-        public string ClientId { get; }
+        public string Identifier { get; }
```

Comment:
A very cryptic name. An identifier is a very overloaded term.

Reply:
This corresponds to the ClientId from the Track 1 library, but we were thinking that ClientId was a bit overloaded based on the ClientId from AAD. 
We are actually planning on making this internal for now, but if we need it to be public we could consider other names.

Reply:
> we were thinking

I really want to be sarcastic here and say "Why not ask the community?" but then it will be bad of me, right? 😉 

The mental jump from SB client to AAD is a substantial leap. I'm not sure this would necessarily be an issue. And even if it would, does any 'client ID' automatically means AAD? Perhaps context is more important? Or an XML document that can explain what it is it if in doubt. Just saying.

Reply:
What do you think in general about not exposing this property as public?

Reply:
For customers that do use message handlers (or message processor in track2 I assume), if it's a single client that is failing, not sure how much value it will provide. But the moment they have more than one processor, that's a different conversation. I've stopped using the built-in message pump and it's not something that would help my scenarios.